sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisClient.cs

```diff
@@ -82,10 +82,13 @@ protected ConversationAnalysisClient()
         /// <exception cref="RequestFailedException">The service returned an error. The exception contains details of the service error.</exception>
         public virtual async Task<Response<AnalyzeConversationTaskResult>> AnalyzeConversationAsync(string utterance, ConversationsProject project, AnalyzeConversationOptions options = null, CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotNull(utterance, nameof(utterance));
+            Argument.AssertNotNullOrEmpty(utterance, nameof(utterance));
```

Comment:
Should also document the `ArgumentException` if `utterance` is an empty string. Same below as well.